juju/testing/conn.go

```diff
@@ -432,7 +432,7 @@ func DefaultVersions(conf *config.Config) []version.Binary {
 	defaultSeries.Add(config.PreferredSeries(conf))
 
 	hostSeries, err := series.HostSeries()
-	if err != nil {
+	if err == nil {
```

Comment:
I looked over this and missed it :)

Reply:
Me too! I guess we're so used to seeing `if err != nil` ... and usually it's correct.